src/utils.cpp

```diff
@@ -1628,6 +1629,179 @@ int get_skill(struct char_data *ch, int skill, int &target)
     }
   }
 
+  // Reflex Recorder, adds +1 die to Physical Skill tests for its associated skill. -Vile
+  // Utilizes Value 6 to reference its associated skill.
+  if (recorder)
+  for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content)
+  if (skill == SKILL_EDGED_WEAPONS && GET_OBJ_VAL(bio, 5) == 1) {
+    snprintf(ENDOF(gskbuf), sizeof(gskbuf) - strlen(gskbuf), "Reflex Recorder Skill Increase: +1. ");
+    skill_dice += 1;
+  }
```

Comment:
I think something went wrong with your copy-paste in this section. The effect of this combined for/if is: "For every piece of bioware you have, add 1 skill if the unchanging skill value is edged weapons." Thus, if someone has 10 pieces of ware and is using the edged weapons skill, they get +10 to that skill.